src/components/Loader/Loader.module.scss

```
@@ -0,0 +1,40 @@
+
+
+.loaderContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.loaderContainer_size_s {
+  width: 24px;
+  height: 24px;
+}
+
+.loaderContainer_size_m {
+  width: 48px;
+  height: 48px;
+}
+
+.loaderContainer_size_l {
+  width: 60px;
+  height: 60px;
+}
```

[+] Для именования классов советую пользоваться БЭМ, это удобно. Например, здесь можно записать так, что чуть более читабельно))

```suggestion
.loaderContainer {
  display: flex;
  justify-content: center;
  align-items: center;
  
  &_size {
    &-s {
      width: 24px;
      height: 24px;
    }
    
    &-m {
      width: 48px;
      height: 48px;
    }
  }
}
```